src/test/java/org/cyclonedx/parsers/JsonParserTest.java

```diff
@@ -155,6 +155,14 @@ public void testParsedObjects12Bom() throws Exception {
         Assert.assertNull(d11.getDependencies());
     }
 
+    @Test
+    public void testValid12BomWithVulnerabilities() throws Exception {
+        final byte[] bomBytes = IOUtils.toByteArray(this.getClass().getResourceAsStream("/bom-1.2-vulnerability.json"));
+        final JsonParser parser = new JsonParser();
+        final Bom bom = parser.parse(bomBytes);
+        Assert.assertNotNull(bom);
```

Comment:
This is mostly a stub test to make sure things occur correctly, it needs to be filled out in much more detail